### ISSUE TITLE
GitHub pull requests are not linked if there is no issue

### ISSUE THREAD

**Issue by garrappachc** [conventional-changelog/conventional-changelog#655]
The generated changelog always links to `/issues/issue-number` even if there is no such issue and the number refers to pull request only.
Example: take a look at the

> deps: update dependency helmet to v3.23.1 (#440) (0e59897)

here: https://github.com/tf2pickup-pl/server/blob/master/CHANGELOG.md#300-2020-06-17
It links to https://github.com/garrappachc/server/issues/440 even though there is no such issue - there is only a pull request with that number: https://github.com/tf2pickup-pl/server/pull/440

I have no idea if this is only a GitHub-related issue.

**Comment by tommywo** [comment]
This is due to # linking to both pull request and issues. Note that `https://github.com/tf2pickup-pl/server/issues/440` leads to correct pull request, only your link has wrong owner.

**Comment by garrappachc** [comment]
You're right. The issue seems to be in linking to the wrong repo, then. I have no clue how to make `conventional-changelog` link the upstream one - making prs from forks seems to be the canonical way.

**Comment by zoubingwu** [comment]
Seems it uses git remote origin by default, you could specify repo url in your package.json instead

```jsonc
{
  //...
  "repository": {
    "url": "https://github.com/tf2pickup-pl",
    "type": "git"
  }
}
```

**Comment by dangreen** [comment]
Set the canonical upstream repository in package.json or through repository(); links will not depend on a remote named origin. Closing as answered.
